scalafix/rules/src/main/scala/fix/Cats_v1_0_0.scala

```diff
@@ -242,6 +242,23 @@ case class RenameTransformersLift(index: SemanticdbIndex)
 
 }
 
+case class RenameApplyApConst(index: SemanticdbIndex)
+  extends SemanticRule(index, "RenameApplyApConst") {
+
+  override def fix(ctx: RuleCtx): Patch =
+    ctx.replaceSymbols(
+      "_root_.cats.Apply.forEffect." -> "apL",
+      "_root_.cats.Apply.followedBy." -> "apR",
+      "_root_.cats.Apply.Ops.forEffect." -> "apL",
+      "_root_.cats.Apply.Ops.followedBy." -> "apR",
+      "_root_.cats.NonEmptyParallel.parForEffect." -> "parApL",
+      "_root_.cats.NonEmptyParallel.parFollowedBy." -> "parApR",
+      "_root_.cats.FlatMap.forEffectEval." -> "apLEval",
+      "_root_.cats.FlatMap.followedByEval." -> "apREval",
+    )
```

Comment:
These rewrite rules still need to be updated to `productR`/`productL`, right?

Reply:
Ah yep, they sure do.  I missed those.

Reply:
fix submitted https://github.com/typelevel/cats/pull/2117